api\api.py:138: error: List comprehension has incompatible type List[HotspotEntry]; expected List[dict[str, Any]]  [misc]
api\api.py:140: error: Value of type "HotspotEntry" is not indexable  [index]
api\api.py:142: error: Argument 1 to "is_within_timestamp_range" has incompatible type "HotspotEntry"; expected "dict[str, Any]"  [arg-type]
api\api.py:160: error: Argument "features" to "SpatialGeoJsonResponse" has incompatible type "list[dict[str, Any]]"; expected "list[Feature]"  [arg-type]
api\api.py:181: error: Incompatible types in assignment (expression has type "HotspotEntry | None", variable has type "dict[str, Any] | None")  [assignment]
api\api.py:193: error: Argument "properties" to "IDGeoJsonResponse" has incompatible type "dict[Never, Never]"; expected "Properties"  [arg-type]
Found 6 errors in 1 file (checked 12 source files)
